### DISCUSSION TITLE
Ajout d'une rubrique "à propos" (bas de page)

### DISCUSSION THREAD

**Discussion by SandrineRobert** [unistra/esup-immersup-demandes#3] [2- Partie publique]
Nous aimerions une rubrique supplémentaire en bas de page, intitulée "à propos", pour pouvoir faire apparaître le financement de la plateforme :
"Projet financé par le SGPI et opéré par la Banque des Territoires dans le cadre du PIA 3 – Territoires d’Innovation Pédagogique, sous l’égide de France 2030"
Avec possibilité d'intégrer les logos sauf si la contrainte technique est trop forte.

**Comment by efinkbeiner5** [comment]
Complexité : faible

**Comment by efinkbeiner5** [comment]
fait en css par les établissements
